packages/react/src/components/Chat/ChatItem.tsx

```diff
@@ -147,6 +147,7 @@ ChatItem.slotClassNames = {
 }
 
 /**
- * A chat item represents a single event in a chat.
+ * A ChatItem is container for Chat item's content.
```

Comment:
Maybe replacing the `Chat item's content` with something like `one Chat row` would be more descriptive

Reply:
have simplified description to 
> A ChatItem is container for single event in Chat (e.g. message, notification, etc).

I think that it is better to avoid visual commitments (like, `row`) in the component's description, to ensure that only component's semantics is provided.